<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="coefsLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Значения при коэф.:</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>251</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Мнимая</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="aReEdit">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>b:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="aImEdit">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Вещественная</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>a:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>c:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cReEdit">
      <property name="text">
       <string>-6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="bReEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="bImEdit">
      <property name="text">
       <string>-5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="cImEdit">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Значение х:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x=</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="xReEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="xImEdit">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="solveButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Решить</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>21</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Полином:</string>
   </property>
  </widget>
  <widget class="QLabel" name="polynomShowLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Не задан</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Корни:</string>
   </property>
  </widget>
  <widget class="QLabel" name="firstRootLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="secondRootLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Решение для заданного х:</string>
   </property>
  </widget>
  <widget class="QLabel" name="solutionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Незаполненные поля приравниваются к 0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
